code/datums/weather/weather_types/acid_rain.dm

```diff
@@ -52,6 +52,10 @@
 	if(prob(max(0,100-resist)))
 		L.adjustFireLoss(7)
 		to_chat(L, span_boldannounce("You feel the acid rain melting you away!"))
+	cleanup(L, TRUE)
+	if(L.on_fire && prob(10))
+		L.ExtinguishMob()
+		to_chat(L, span_warning("As the acidic rain burns your body away, it also extinguishes the fire with a steamy hiss."))
```

Comment:
```suggestion
	if(L.fire_stacks > -20)
		L.fire_stacks = max(-20, L.fire_stacks - 3)
```
Something like this. Also side note, why the hell is normal rain a child of acid rain?